lib/identity_config.rb

```diff
@@ -64,7 +64,7 @@ def add(key, type: :string, allow_nil: false, enum: nil, options: {})
       raise "unexpected #{key}: #{value}, expected one of #{enum}"
     end
 
-    @written_env[key] = converted_value
+    @written_env[key] = converted_value.freeze
```

Comment:
😂 oh no there goes my `.reverse!` trick in the console to test flipping regions

Reply:
oh whoops, that was not related

Reply:
for what it's worth, the config is only instantiated once, so I'm not sure how useful it is performance-wise to freeze in this case.  it could be useful for preventing incidental configuration mutation in the running application though 😀